### PR TITLE
Added Contributor

### LINKED ISSUE #34: Adding Contributor section in readme  ( Automatically )
I will like to add a feature this will update contributor list in read me automatically !! Will start to work on this issue as soon as i get assigend !!

### PR DESCRIPTION
**Pull request by rohan-kulkarni-25**
I have added the automatically updating contributor in read me !! 
Fixes #34

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,3 +49,18 @@ This would be an enriching experience for all Student Developers, Project Manage
 ## License📜
 
 [MIT License](https://github.com/HarshCasper/SpeakEd/blob/main/LICENSE)
+
+## Contributor
+ 
+### Credit goes to these people:✨
```
Comment (HarshCasper) [inline README.md]: Remove this

File: README.md [hunk 2]
```diff
@@ -49,3 +49,18 @@ This would be an enriching experience for all Student Developers, Project Manage
 ## License📜
 
 [MIT License](https://github.com/HarshCasper/SpeakEd/blob/main/LICENSE)
+
+## Contributor
```
Comment (HarshCasper) [inline README.md]: Make it `Contributors`

File: README.md [hunk 3]
```diff
@@ -49,3 +49,18 @@ This would be an enriching experience for all Student Developers, Project Manage
...
+ 
+### Credit goes to these people:✨
+
+<table>
+	<tr>
+		<td>
+			<a href="https://github.com/HarshCasper/SpeakEd/graphs/contributors">
+  <img src="https://contrib.rocks/image?repo=HarshCasper/SpeakEd" />
```
Comment (HarshCasper) [inline README.md]: Indent this

File: README.md [hunk 4]
```diff
@@ -49,3 +49,18 @@ This would be an enriching experience for all Student Developers, Project Manage
...
+</a>
+		</td>
+	</tr>
+</table>
+
```
Comment (HarshCasper) [inline README.md:62]: Remove this line

**Review by HarshCasper — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (rohan-kulkarni-25) [reply README.md:62]: Done suggested changes have a look !!

**Review by HarshCasper — APPROVED** [review]

**Pull request merged**